### PR TITLE
use junit platform

### PR DESCRIPTION
**Pull request by santitigaga**
Before submitting a pull request, please read
https://github.com/bancolombia/scaffold-clean-architecture/wiki/Contributing

## Category
- [ ] Feature
- [x] Fix

## Checklist
- [ ] The pull request is complete according to the [guide of contributing](https://github.com/bancolombia/scaffold-clean-architecture/wiki/Contributing)
- [ ] Automated tests are written
- [ ] The documentation is up-to-date
- [ ] The pull request has a descriptive title that describes what has changed, and provides enough context for the changelog
- [ ] If the pull request has changed structural files, you have implemented an UpgradeAction according to the [guide](https://github.com/bancolombia/scaffold-clean-architecture/wiki/Contributing#upgradeaction)

### CODE REVIEW

**Review by juancgalvis — CHANGES_REQUESTED** [review]
Hi, as you are changing the `main.gradle` file you should implement an `UpgradeAction`

**Review by juancgalvis — APPROVED** [review]
Thanks

File: src/main/java/co/com/bancolombia/factory/upgrades/UpgradeMainGradle.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import static co.com.bancolombia.Constants.MainFiles.MAIN_GRADLE;
+
+import co.com.bancolombia.factory.ModuleBuilder;
+import java.util.concurrent.atomic.AtomicBoolean;
+import lombok.SneakyThrows;
+
+public class UpgradeMainGradle implements UpgradeAction {
+
+  @Override
+  public boolean up(ModuleBuilder builder) {
```
Comment (juancgalvis) [inline src/main/java/co/com/bancolombia/factory/upgrades/UpgradeMainGradle.java]: Are you using this default implementation? If not, you can delete it and make this class abstract.

Comment (santitigaga) [reply src/main/java/co/com/bancolombia/factory/upgrades/UpgradeMainGradle.java]: you are right, solved

**Review by juancgalvis — APPROVED** [review]
Thats's ok

**Pull request merged**